tools/common-code/cmonitor_statistics_engine.py

```diff
@@ -102,16 +103,21 @@ def dump_cpu_stats(self, verbose) -> dict:
     def dump_cpu_throttle_stats(self, verbose) -> dict:
         return self.cpu_throttle.dump_json(verbose)
 
-    def insert_memory_stats(self, stats: dict, sample_index: int) -> None:
-        if "stat.rss" in stats:
-            self.memory.insert_stat(stats["stat.rss"])
+    def insert_memory_stats(self, stats: dict, sample_index: int, cgroup_version: int) -> None:
+        stat_label = "stat.rss" if cgroup_version == 1 else "stat.file"
```

Comment:
when we are dealing with cgroup_version==2, I guess we should be using the "file" + "anon" metric, if we want to have something similar to the cgroupv1 "rss" metric... what do you think?